.github/workflows/stable.yml

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2022 Sharezone UG (haftungsbeschränkt)
+# Licensed under the EUPL-1.2-or-later.
+#
+# You may obtain a copy of the Licence at:
+# https://joinup.ec.europa.eu/software/page/eupl
+#
+# SPDX-License-Identifier: EUPL-1.2
+
+name: stable
+
+concurrency:
+  # Our iOS builds require unique build numbers. Therefore, we should never
+  # build a new version while a build is already running. This is why we use the
+  # branch name as a concurrency group. This way, we can only build one version
+  # per branch at the same time.
+  group: app-release
+
+on:
+  workflow_dispatch:
+    inputs:
+      ios-changelog:
+        type: string
+        description: |
+          The changelog for the iOS release. This will be used as the release
+          notes in the App Store release.
+        required: true
```

Comment:
Or should we add an option that you can select which platforms should get a new stable release? Imagine that we fix a web only bug and we want only deploy web.

Reply:
Yes, that sounds like a nice idea. Good thinking! :)

Reply:
I have changed the description to:

> iOS changelog: Used for the App Store release notes. If the changelog is not provided, the deployment for iOS will be skipped.

Assuming that if there is no changelog, nothing has changed for iOS and no new deployment is needed.